CMSgov/easi-app
# [EASI-2518] Emails when assigning TRB lead

# EASI-2518

## Changes and Description

- Added an email that is sent to the TRB lead when they are assigned to a TRB request
- Updated the existing email that is sent to the TRB team when the lead is assigned so that both emails are sent in parallel
- Added related tests

## How to test this change
Make a TRB request, assign a lead, then navigate to localhost:1080 and verify that the two emails were sent as expected:

```GraphQL
# Make a TRB request and get it to a proper state
mutation {
  createTRBRequest(requestType: NEED_HELP) {
    id
    type
  }
}

# Add a lead
mutation {
  updateTRBRequestTRBLead(
    input: { trbLead: "ABCD", trbRequestId: "trbIdFromAbove" }
  ) {
    id
    trbLead
  }
}
```
![image](https://user-images.githubusercontent.com/95702411/219383254-278d507f-cbd7-4134-a84c-c3324a5470e6.png)


## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
